### DISCUSSION TITLE
Error TextColum Align Center with ViewAction and EditAction

### DISCUSSION THREAD

**Discussion by moskoweb** [filamentphp/filament#4898] [General]
After the last update, when adding the Edit and View action in the list of items table of a relationship, it ends up aligning the TextColums to the center, even without the alignment method.

1. Create a Resource with a relationship.
2. Add a column of type TextColumn.
3. Add the ViewAction or EditAction action.
<img width="1365" alt="Captura de Tela 2022-11-11 às 00 26 03" src="https://user-images.githubusercontent.com/6052272/201256499-ebf8ace1-525c-413a-9b97-d91a28ff885f.png">
<img width="1356" alt="Captura de Tela 2022-11-11 às 00 16 01" src="https://user-images.githubusercontent.com/6052272/201256504-8f965599-5123-4b0e-9c3e-3adc737ae945.png">

**Comment by zepfietje** [comment]
Have you recompiled your assets, @moskoweb?

**Reply by moskoweb** [reply]
I've already done that, but the same error remains, if you add the Edit or View action, simply the text columns are aligned to the center, if you remove, they return to the default on the left.

 And even if you put the alignment method, it is ignored.

**Reply by zepfietje** [reply]
I can't reproduce this. Could you set up a reproduction repository or create an issue and fill out the entire bug report template?

**Reply by danharrin** [reply]
I bet you just need to recompile your theme... we switched from `text-left` to `text-start`

**Reply by moskoweb** [reply]
I've already recompiled it, and it's still the same, it's only when you put the Edit and View actions that this occurs, of centralizing.

**Reply by danharrin** [reply]
Is the text-start class present in your compiled theme stylesheet?

**Reply by moskoweb** [reply]
I use the default Filament theme, still creating the assets before customizing anything.

**Reply by zepfietje** [reply]
You customized the color so you are using a custom theme in fact. If you're totally sure that you've compiled your assets, there's nothing we can do without looking at a reproduction repository.

**Reply by moskoweb** [reply]
I managed to fix it, deleted the vendor and reinstalled everything, then it worked.

**Reply by danharrin** [reply]
so the alignment is back to left?

**Reply by moskoweb** [reply]
yes

**Reply by danharrin** [reply]
🤔 some caching issue then. weird
